src/overlays/actors/ovl_Obj_Kendo_Kanban/z_obj_kendo_kanban.c

```diff
@@ -217,238 +245,248 @@ void ObjKendoKanban_Destroy(Actor* thisx, PlayState* play) {
                                this->actor.home.pos.x - 5.0f, this->actor.home.pos.y, this->actor.home.pos.z, 0, 0, 0,
-                               0xA);
+                               LEFT_HALF);
+
+            this->cornerPoints[0] = sPointTC;
+            this->cornerPoints[1] = sPointTR;
+            this->cornerPoints[2] = sPointBR;
+            this->cornerPoints[3] = sPointBC;
 
-            this->unk_29C[0] = D_80B666DC;
-            this->unk_29C[1] = D_80B666E8;
-            this->unk_29C[2] = D_80B66730;
-            this->unk_29C[3] = D_80B66724;
         } else {
-            this->unk_30C = (OBJKENDOKANBAN_F_8 | OBJKENDOKANBAN_F_4);
-            this->unk_304 = -0x71C;
-            this->actor.velocity = D_80B66678;
-            this->unk_2CC = D_80B666A8;
+            // Horizontal cuts initialize the bottom half, spawn the top half.
+            this->boardFragments = BOTTOM_HALF;
+            this->rotationVelocity = -0x71C; // -10 degrees
+            this->actor.velocity = sVelocityBottomHalf;
+            this->centerPoint = sCenterPointBottomHalf;
 
             Actor_SpawnAsChild(&play->actorCtx, &this->actor, play, ACTOR_OBJ_KENDO_KANBAN, this->actor.home.pos.x,
-                               this->actor.home.pos.y + 5.0f, this->actor.home.pos.z, 0, 0, 0, 3);
+                               this->actor.home.pos.y + 5.0f, this->actor.home.pos.z, 0, 0, 0, TOP_HALF);
 
-            this->unk_29C[0] = D_80B666F4;
-            this->unk_29C[1] = D_80B6670C;
-            this->unk_29C[2] = D_80B66730;
-            this->unk_29C[3] = D_80B66718;
+            this->cornerPoints[0] = sPointCL;
+            this->cornerPoints[1] = sPointCR;
+            this->cornerPoints[2] = sPointBR;
+            this->cornerPoints[3] = sPointBL;
         }
-    } else if (this->unk_30C == (OBJKENDOKANBAN_F_8 | OBJKENDOKANBAN_F_2)) {
-        this->unk_304 = 0x71C;
-        this->actor.velocity = D_80B66660;
-        this->unk_2CC = D_80B66690;
-
-        this->unk_29C[0] = D_80B666D0;
-        this->unk_29C[1] = D_80B666DC;
-        this->unk_29C[2] = D_80B66724;
-        this->unk_29C[3] = D_80B66718;
-    } else if (this->unk_30C == (OBJKENDOKANBAN_F_2 | OBJKENDOKANBAN_F_1)) {
-        this->unk_304 = 0x71C;
-        this->actor.velocity = D_80B66684;
-        this->unk_2CC = D_80B666B4;
-
-        this->unk_29C[0] = D_80B666D0;
-        this->unk_29C[1] = D_80B666E8;
-        this->unk_29C[2] = D_80B6670C;
-        this->unk_29C[3] = D_80B666F4;
+
+    } else if (this->boardFragments == LEFT_HALF) {
+        // Initialize the newly spawned left half
+        this->rotationVelocity = 0x71C; // 10 degrees
+        this->actor.velocity = sVelocityLeftHalf;
+        this->centerPoint = sCenterPointLeftHalf;
+
+        this->cornerPoints[0] = sPointTL;
+        this->cornerPoints[1] = sPointTC;
+        this->cornerPoints[2] = sPointBC;
+        this->cornerPoints[3] = sPointBL;
+
```

Comment:
We don't usually put the extra space at the end of the ifs, but it doesn't really matter.